### DISCUSSION TITLE
Welcome to fDOG Discussions!

### DISCUSSION THREAD

**Discussion by trvinh** [BIONF/fDOG#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lfp-a** [comment]
I ran into a BUG
When I submit a task, he will report an error ：
TERM not set at /datapool//miniconda3/lib/python3.7/site-packages/fdog/bin/oneSeq.pl line 138.

I don't know how to fix it 
Can someone take a look at it?  
thanks

**Reply by trvinh** [reply]
Hi,
Could you please tell us how did you run fDOG, it would be helpful :) 
Best,
Vinh
